src/core/tests/pattern.cpp

```diff
@@ -510,37 +510,35 @@ TEST(pattern, matching_optional) {
                          std::make_shared<op::v0::Abs>(c)));
 }
 
-TEST(pattern, optional_full_match) {
+TEST(pattern, DISABLED_optional_full_match) {
```

Comment:
it's better to use some skip list file and mention the jira ticket

Reply:
As I can see, these tests were not configured to work with a skip list files
Ok, lets leave it as it is, but please add a comment to each test about the issue description and a jira ticket